### PR TITLE
publish for nodejs target

### PR DESCRIPTION
**Pull request by kristifidani**
# Pull Request Template

## Motivation and Context

The were publishing the wasm bindings for web target. But then we could not create a quickstart project for typescript. Therefore, we need to deploy for the nodejs target.

## Summary

Provide a short summary of changes in this pull request.

- Update the deploy script to use the nodejs target.

## Checklist

Please ensure that your PR meets the following requirements:

### General

- [ ] Code follows project style and guidelines.
- [ ] No redundant or duplicate code.
- [ ] No added new dependencies without clear justification and approval.

### Documentation

- [ ] Added/updated relevant documentation (e.g., README, inline comments).
- [ ] Added a CHANGELOG entry for major changes.

### Testing

- [ ] Added relevant test cases, leveraging tools like `rstest` or similar for reusable tests.
- [ ] Tests pass locally.

### Build & CI/CD

- [ ] Confirmed no CI/CD pipeline issues.
- [ ] Updated build scripts where necessary.

## Comments to Reviewer

Please provide any comments or points of consideration for the reviewers, such as specific areas of focus, potential edge cases, or context that would aid the review process.

### Reviewer Responsibilities

- [ ] Code adheres to style and guidelines.
- [ ] All tests are appropriately covered and they pass.
- [ ] Consider performance and/or security impacts.
- [ ] Constructive feedback is provided.

### CODE REVIEW

**Review by antbern — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -176,10 +176,10 @@ jobs:
       - name: Install wasm-pack
         run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
 
-      - name: Build wasm sdk for web
+      - name: Build wasm sdk for nodejs
         run: |
           cd sdk/bindings/wasm
-          wasm-pack build --release --scope eto --target web --out-dir out/web/pkg --out-name cryptpay-sdk-web
+          wasm-pack build --release --scope eto --target nodejs --out-dir out/web/pkg --out-name cryptpay-sdk-node
```
Comment (antbern) [inline .github/workflows/ci.yml:182]:
```suggestion
          wasm-pack build --release --scope cawaena --target nodejs --out-dir out/web/pkg --out-name cryptpay-sdk-node
```

**Review by antbern — APPROVED** [review]

**Review by lszetg — APPROVED** [review]
👍
